<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsdEngineClass</class>
 <widget class="QMainWindow" name="AsdEngineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AsdEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>872</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>外挂练习</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,20,1">
                   <item>
                    <widget class="QSplitter" name="splitter">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <widget class="QPushButton" name="Button_changelua1">
                      <property name="maximumSize">
                       <size>
                        <width>60</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>LUA窗口1</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="Button_changelua2">
                      <property name="maximumSize">
                       <size>
                        <width>60</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>LUA窗口2</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="Button_changelua3">
                      <property name="maximumSize">
                       <size>
                        <width>60</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>LUA窗口3</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="Button_changelua4">
                      <property name="maximumSize">
                       <size>
                        <width>60</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>LUA窗口4</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="Button_changelua5">
                      <property name="maximumSize">
                       <size>
                        <width>60</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>LUA窗口5</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="Button_changelua6">
                      <property name="maximumSize">
                       <size>
                        <width>60</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>LUA窗口6</string>
                      </property>
                     </widget>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="widgetLua" native="true"/>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                     <item>
                      <widget class="QComboBox" name="comboBox"/>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioButton">
                       <property name="text">
                        <string>X64</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioButton_2">
                       <property name="text">
                        <string>X86</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_3">
                       <property name="text">
                        <string>标准64Call</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <layout class="QFormLayout" name="formLayout">
                       <item row="2" column="1">
                        <widget class="QPushButton" name="fastLuaButton4">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>Push</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="1">
                        <widget class="QPushButton" name="fastLuaButton2">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>Push</string>
                         </property>
                        </widget>
                       </item>
                       <item row="3" column="1">
                        <widget class="QPushButton" name="fastLuaButton6">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="4" column="1">
                        <widget class="QPushButton" name="fastLuaButton8">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="4" column="0">
                        <widget class="QPushButton" name="fastLuaButton7">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="3" column="0">
                        <widget class="QPushButton" name="fastLuaButton5">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="0">
                        <widget class="QPushButton" name="fastLuaButton3">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QPushButton" name="fastLuaButton1">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>回血2</string>
                         </property>
                        </widget>
                       </item>
                       <item row="5" column="0">
                        <widget class="QPushButton" name="fastLuaButton9">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="5" column="1">
                        <widget class="QPushButton" name="fastLuaButton10">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="6" column="0">
                        <widget class="QPushButton" name="fastLuaButton11">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                       <item row="6" column="1">
                        <widget class="QPushButton" name="fastLuaButton12">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>PushButton</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QPushButton" name="Tab1CloseLua">
                       <property name="text">
                        <string>停止</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="Tab1Runlua">
                       <property name="text">
                        <string>执行</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>ETW监控</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,4">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QFrame" name="frame_3">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <widget class="QPushButton" name="pushButton_4">
                      <property name="geometry">
                       <rect>
                        <x>70</x>
                        <y>0</y>
                        <width>71</width>
                        <height>20</height>
                       </rect>
                      </property>
                      <property name="text">
                       <string>清空</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>71</width>
                        <height>20</height>
                       </rect>
                      </property>
                      <property name="text">
                       <string>运行驱动</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="pushButton_5">
                      <property name="geometry">
                       <rect>
                        <x>140</x>
                        <y>0</y>
                        <width>80</width>
                        <height>20</height>
                       </rect>
                      </property>
                      <property name="text">
                       <string>过滤规则</string>
                      </property>
                     </widget>
                     <widget class="QRadioButton" name="radioButton_3">
                      <property name="geometry">
                       <rect>
                        <x>630</x>
                        <y>0</y>
                        <width>91</width>
                        <height>18</height>
                       </rect>
                      </property>
                      <property name="text">
                       <string>锁界面</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="pushButton_6">
                      <property name="geometry">
                       <rect>
                        <x>220</x>
                        <y>0</y>
                        <width>80</width>
                        <height>20</height>
                       </rect>
                      </property>
                      <property name="text">
                       <string>保存</string>
                      </property>
                     </widget>
                     <widget class="QPushButton" name="pushButton_7">
                      <property name="geometry">
                       <rect>
                        <x>300</x>
                        <y>0</y>
                        <width>80</width>
                        <height>20</height>
                       </rect>
                      </property>
                      <property name="text">
                       <string>开始</string>
                      </property>
                      <property name="icon">
                       <iconset theme="start">
                        <normaloff>C:/Users/Administrator/.designer/backup</normaloff>C:/Users/Administrator/.designer/backup</iconset>
                      </property>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QTableWidget" name="tableWidget">
                     <column>
                      <property name="text">
                       <string>编号</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>进程名</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>PID</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>函数</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>备用</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Rcx</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Rdx</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>R8</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>R9</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>其他</string>
                      </property>
                     </column>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
           <widget class="QPushButton" name="TableButton">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>表格窗口</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
        <item>
         <widget class="QTextEdit" name="txtLog"/>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_4">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AsdEngine.qrc"/>
 </resources>
 <connections/>
</ui>
